dev/com.ibm.ws.jsfContainer_fat_2.3/.project

```diff
@@ -20,4 +20,15 @@
 		<nature>org.eclipse.jdt.core.javanature</nature>
 		<nature>bndtools.core.bndnature</nature>
 	</natures>
+	<filteredResources>
+		<filter>
+			<id>1718202868445</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
```

Comment:
What are we doing here? I think this can likely be removed?

Reply:
Not sure, I think it was auto generated